Mathlib/NumberTheory/FunctionField.lean

```diff
@@ -269,11 +269,11 @@ instance valuedFqtInfty : Valued (FqtInfty Fq) ℤₘ₀ :=
 set_option linter.uppercaseLean3 false in
 #align function_field.valued_Fqt_infty FunctionField.valuedFqtInfty
 
-theorem valuedFqtInfty.def {x : FqtInfty Fq} :
+theorem valuedFqtInfty.def' {x : FqtInfty Fq} :
```

Comment:
```suggestion
-- Adaptation note: 2024-03-15
-- Renamed to avoid the reserved name `valuedFqtInfty.def`.
theorem valuedFqtInfty.def' {x : FqtInfty Fq} :
```